book/http_cache.rst

```diff
@@ -1031,21 +1064,36 @@ matter), Symfony uses the standard ``render`` helper to configure ESI tags:
 
     .. code-block:: jinja
 
+        {# app/Resources/views/about.html.twig #}
```

Comment:
I don't have a really strong opinion about it. However, I don't like using `default` as sub-directory. I also don't really like DefaultController. Both are not default, they are as not default as any other template and controller. What do you think about using `statics` and `StaticController`